packaging_automation/upload_to_package_cloud.py

```diff
@@ -13,6 +13,7 @@
     "el/7": 140,
     "el/8": 205,
     "ol/7": 146,
+    "debian/bullseye": 207,
```

Comment:
package cloud distrio definition is as below       
 {
          "id": 207,
          "display_name": "11 bullseye",
          "index_name": "bullseye",
          "version_number": "11.0"
        },

Reply:
thank you for sharing